### DISCUSSION TITLE
dynamically pass the stream key through Push Publishing

### DISCUSSION THREAD

**Discussion by Bistiara** [OvenMediaLabs/OvenMediaEngine#1711] [Help] [answered]
Good afternoon, 
I'm trying to broadcast from one ome to another ome using push publish, I used the configuration from your documentation
https://airensoft.gitbook.io/ovenmediaengine/0.16.8/push-publishing

could you please tell me if it is possible to dynamically pass the stream key through Push Publishing map.xml? I see this in the documentation, but in the server logs, I see the name of the variable as text.

```
<?xml version="1.0" encoding="UTF-8"?>
<PushInfo>
  <Push>
    <!-- [Must] -->
    <Enable>true</Enable>
    <!-- [Must] -->
    <StreamName>stream*</StreamName>
    <!-- [Must] -->
    <Protocol>rtmp</Protocol>
    <!-- [Must] -->
    <Url>rtmp://ome:1935/app/${StreamName}</Url>
  </Push>
```

${StreamName} - does not work 

https://prnt.sc/ik2c3fCNX66S

**Comment by Keukhan** [comment, marked as answer]
@Bistiara 

Thank you for reporting.
There was a problem with the macro-related guide in the document. Please change it as below and use it.

- ${SourceStream} : Source Stream Name
- ${Stream} : Output Stream Name
